### PR TITLE
doc change for setting up environment in Ubuntu

### PR DESCRIPTION
**Pull request by karmadolkar**
#3922 @cverna

### CODE REVIEW

File: docs/developer/vagrant.rst [hunk 1]
```diff
@@ -11,6 +11,9 @@ get started, simply use these commands::
     $ sudo systemctl enable libvirtd
     $ sudo systemctl start libvirtd
 
+If you're on Ubuntu, use this in place of the first command::
```
Comment (cverna) [inline docs/developer/vagrant.rst:14]:
```suggestion
On distribution using a Debian base like Ubuntu, use this in place of the first command::
```

File: docs/developer/vagrant.rst [hunk 2]
```diff
@@ -11,6 +11,9 @@ get started, simply use these commands::
...
+		$ sudo apt install ansible libvirt-dev vagrant-libvirt vagrant-sshfs
```
Comment (cverna) [inline docs/developer/vagrant.rst:15]: The indentation seems a little bit off, I think that it should be 4 spaces to look like the rest of the document

**Review by cverna — COMMENTED** [review]
Thanks, just couple comments but looks good :+1:

**Comment by cverna** [comment]
@karmadolkar Could you squash you commits to a single commit ? (https://gist.github.com/patik/b8a9dc5cd356f9f6f980#the-harder-and-less-flexible-way). 

Also you will need to sign off you commit using the `-s` option in the git commit command `git commit -s -m "This is signed off commit message `

**Comment by accakks** [comment]
Hi @karmadolkar also an outreachy applicant here, this was really helpful for me as well, thankyou.

@cverna @karmadolkar However while using 2nd command, I got error
`Failed to enable unit: Unit file libvirtd.service does not exist.`

Installing virt-manager fixed this for me. 
```
sudo apt install virt-manager
sudo virt-manager

```
I understand we might not wanna give out instructions on installing everything, but we can add a heads up? (Example change below in bold)

> Before you get started, ensure that your host machine has virtualization extensions enabled in its BIOS so the guest doesn’t go slower than molasses. **Also make sure you have virt-manager installed** To get started, simply use these commands: ........

@cverna Let me know if this looks like something to add? 
(Because terminal message doesn't say that 'try installing vert-manager')

**Review by cverna — APPROVED** [review]

**Pull request merged**